lib/core/src/Cardano/Wallet/Primitive/Types.hs

```diff
@@ -699,26 +699,29 @@ data TransactionInfo = TransactionInfo
 -- function where @x@ can be the transaction size in bytes or, a number of
 -- inputs + outputs.
 --
--- @a@ and @b@ are constant coefficients.
-data FeePolicy =
-    LinearFee (Quantity "lovelace" Double) (Quantity "lovelace/x" Double)
+-- @a@, @b@ and @c@ are constant coefficients.
+data FeePolicy = LinearFee
+    (Quantity "lovelace" Double)
+    (Quantity "lovelace/x" Double)
+    (Quantity "lovelace" Double)
     deriving (Eq, Show, Generic)
 
 instance NFData FeePolicy
 
 instance ToText FeePolicy where
-    toText (LinearFee (Quantity a) (Quantity b)) =
-        toText a <> " + " <> toText b <> "x"
+    toText (LinearFee (Quantity a) (Quantity b) (Quantity c)) =
+        toText a <> " + " <> toText b <> "x + " <> toText c
 
 instance FromText FeePolicy where
     fromText txt = case T.splitOn " + " txt of
-        [a, b] | T.takeEnd 1 b == "x" -> LinearFee
+        [a, b, c] | T.takeEnd 1 b == "x" -> LinearFee
             <$> fmap Quantity (fromText a)
             <*> fmap Quantity (fromText (T.dropEnd 1 b))
+            <*> fmap Quantity (fromText c)
         _ ->
             Left $ TextDecodingError
                 "Unable to decode FeePolicy: \
-                \Linear equation not in expected format: a + bx"
+                \Linear equation not in expected format: a + bx + c"
```

Comment:
`c` is not a constant here. I presume we need to multiply with some new variable `y` or `certs`.

I wonder where this FromText is used actually…

Reply:
it is used for example in PersistentField instance https://github.com/input-output-hk/cardano-wallet/blob/master/lib/core/src/Cardano/Wallet/DB/Sqlite/Types.hs#L151